src/repo/github.js

```diff
@@ -93,7 +94,7 @@ const getUserInfo = function(username, hostname, optionalPrivateToken) {
     .then(res => {
       const body = JSON.parse(res.body)
 
-      let profile = body.blog || body.html_url
+      let profile = utilUrl.isValidHttpUrl(body.blog) ? body.blog : body.html_url
```

Comment:
I think it would be better to destructure the export, that way that line would say:
```suggestion
let profile = isValidHttpUrl(body.blog) ? body.blog : body.html_url
```
With the suggested import mentioned above.